index_v2.html

```diff
@@ -23,13 +23,16 @@
 	
 	<div class="sidebar">Sidebar</div>
 	
-	<div class="content">
+		<div class="content">
 		<div class="scroll-area">
 		
 		<h1><center>Login Page</center></h1>
 		
-		<h3>Sub Heading</h3>
+		<h3>Please Login using the following format:</h3>
 		<hr></hr>
+		h3>Username: DFoulds</h3>

```

Comment:
Line 33 I added in a field to collect the Users Username so they can login to my Music Database.